.github/workflows/nintendo.yaml

```diff
@@ -22,6 +22,45 @@ jobs:
           make -f Makefile.3ds
           popd
 
+  _3ds-cia:
+    name: Nintendo 3DS cia
+    runs-on: ubuntu-latest
+    container: devkitpro/devkitarm
+    steps:
+      - name: Clone Project
+        uses: actions/checkout@v2
+
+      - name: Setup cxitool and build
+        shell: bash
+        run: |
+          apt-get update && DEBIAN_FRONTEND=non-interactive apt-get install -y build-essential automake autoconf autotools-dev
+          git clone -b cxi-stuff https://github.com/devkitPro/3dstools/
+          pushd 3dstools/
+          ./autogen.sh
+          ./configure
+          make install
+          popd
+
+      - name: Setup makerom 
+        shell: bash
+        run: |
+          apt-get update && DEBIAN_FRONTEND=non-interactive apt-get install -y build-essential libmbedtls-dev liblzma-dev libyaml-dev
+          git clone https://github.com/3DSGuy/Project_CTR/
+          pushd Project_CTR/makerom
+          make deps
+          make
+          cp -rv bin/makerom /usr/local/bin
+          popd
+
+      - name: Build
+        shell: bash
+        run: |
+          pushd src/
+          make -f Makefile.3ds
+          cxitool --name angband angband.3dsx angband.cxi
+          makerom -v -f cia -o angband.cia -target t -i angband.cxi:0:0 -ignoresign -icon icon.smdh
```

Comment:
The icon should be in `src/nds/icon.smdh` if anything.

Reply:
Done [4efabe6](https://github.com/angband/angband/pull/5376/commits/4efabe63c42fda1dc5779b6d6d79651a115c5282)